app/src/main/java/io/getstream/chat/android/core/poc/library/rest/BanUserRequest.kt

```diff
@@ -1,10 +1,26 @@
 package io.getstream.chat.android.core.poc.library.rest
 
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
 
-class BanUserRequest(
-    val target_user_id: String,
-    val timeout: Int,
-    var reason: String,
-    var channelType: String,
-    var channelId: String
+
+data class BanUserRequest(
+    @SerializedName("target_user_id")
+    @Expose
+    var targetUserId: String,
+
+    @SerializedName("timeout")
+    @Expose
+    var timeout: Int? = null,
+
+    @SerializedName("reason")
```

Comment:
There is no need to repeat field name with @SerializedName if it's already correct.